cockatrice/src/window_main.cpp

```diff
@@ -296,6 +297,9 @@ void MainWindow::serverTimeout()
 void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32 endTime)
 {
     switch (r) {
+        case Response::RespClientUpdateRequired:
+            QMessageBox::critical(this, tr("Error"), tr("Your client does not contain features that the server requires, please update your client and try again."));
```

Comment:
So I started to look into this.  Can either @Daenyth , @ctrlaltca or @poixen tell me if you can pass an entire PB response container through a signal?  Right now the signal to the windows_main function that does the notification is only sending the response code, reason and end time pb container items and wasnt sure if its possible to send a repeated pb string in the container through a signal.  So Ive attempted to send the entire container but am having no luck.